keras2onnx/ke2onnx/bidirectional.py

```diff
@@ -146,19 +149,69 @@ def convert_bidirectional(scope, operator, container):
     else:
         lstm_input_names.append('')  # the name of a non-existing optional variable is an empty string
 
+    oopb = OnnxOperatorBuilder(container, scope)
+    input_shape_tensor = oopb.add_node('Shape',
+                                       [operator.input_full_names[0]],
+                                        operator.inputs[0].full_name + '_input_shape_tensor')
+
+    if container.target_opset >= 10:
+        batch_indices_tensor = oopb.add_node('Slice',
+                                             [input_shape_tensor,
+                                              ('_start', oopb.int64, np.array([0], dtype='int64')),
+                                              ('_end', oopb.int64, np.array([1], dtype='int64')),
+                                              ('_axes', oopb.int64, np.array([0], dtype='int64'))
+                                              ],
+                                             operator.inputs[0].full_name + '_batch_indices_tensor')
+
+        seq_len_tensor = oopb.add_node('Slice',
+                                      [input_shape_tensor,
+                                       ('_start', oopb.int64, np.array([1], dtype='int64')),
+                                       ('_end', oopb.int64, np.array([2], dtype='int64')),
+                                       ('_axes', oopb.int64, np.array([0], dtype='int64'))
+                                       ],
+                                      operator.inputs[0].full_name + '_seq_len_tensor')
+    else:
+        attrs = {'starts': [0], 'ends': [1], 'axes': [0]}
+        batch_indices_tensor = oopb.add_node('Slice',
+                                             [input_shape_tensor],
+                                             operator.inputs[0].full_name + '_batch_indices_tensor', **attrs)
+
+        attrs = {'starts': [1], 'ends': [2], 'axes': [0]}
+        seq_len_tensor = oopb.add_node('Slice',
+                                       [input_shape_tensor],
+                                       operator.inputs[0].full_name + '_seq_len_tensor', **attrs)
```

Comment:
attrs could be added as '..., starts=[1], ends=[2])

Reply:
good suggestion, will do it in future PRs.